build/scripts/build-client-binary

```diff
@@ -27,8 +27,10 @@ gobuild_component() {
         echo skip build
         return
     fi
-    if [ ${component} = 'pmm-admin' -o ${component} = 'pmm-agent' ]; then
-        build_command="cd /tmp/go/src/${extract_path} && make release"
+    if [ ${component} = 'pmm-admin' ]; then
+        build_command="cd /tmp/go/src/${extract_path}/admin && make release"
+    elif [ ${component} = 'pmm-agent' ]; then
+        build_command="cd /tmp/go/src/${extract_path}/agent && make release"
```

Comment:
looks like we are calling gobuild_component for these components incorrectly here https://github.com/percona/pmm/pull/1126/files#diff-d73461fa7f39050da15a5425872480d2f0e377753a7a1a502b715c42f8f79d4dR110-R111
We don't use component_path at all and I think that it could be used the same way as for percona-toolkit.
Also, we can override extract_path there.

Reply:
It seems that you added wrong link, in points to your comment

Reply:
line 110-111 in this file where we call `gobuild_component`.
https://github.com/percona/pmm/blob/d74fa2b31add84840df0cddc313d91549c6f135d/build/scripts/build-client-binary#L110-L111

Reply:
Done